crates/router/src/core/webhooks/outgoing.rs

```diff
@@ -457,13 +473,50 @@ fn raise_webhooks_analytics_event(
         .and_then(api::OutgoingWebhookContent::get_outgoing_webhook_event_content)
         .or_else(|| get_outgoing_webhook_event_content_from_event_metadata(event.metadata));
 
+    // Fetch updated_event from db
+    let updated_event = state
+        .store
+        .find_event_by_merchant_id_event_id(
+            key_manager_state,
+            &merchant_id,
+            &event_id,
+            merchant_key_store,
+        )
+        .await
+        .attach_printable_lazy(|| format!("event not found for id: {}", &event_id))
+        .map_err(|e| {
+            logger::error!("{:?}", e);
+            e
+        })
```

Comment:
```suggestion
        .map_err(|error| {
            logger::error!(?error);
            error
        })
```

We use the `tracing` crate for logging, so we can use the `?` to use the `Debug` impl for logging. Refer to the [`tracing` docs](https://docs.rs/tracing/0.1.40/tracing/#recording-fields) for more information.

Reply:
Nice! Got it!